assets/style.css

```diff
@@ -13,3 +13,16 @@ p + p {
 .nuxt-scroll-indicator {
   @apply bg-gradient-to-r to-secondary from-blue-700;
 }
+
+/* We use this color just one time */
+.light {
```

Comment:
I think it's easier to use tailwind syntax here, for example:
```css
::selection: {
	@apply bg-[#d4b5d4] dark:bg-primary
}
```

Reply:
Thanks for your comment. I just realized that if we want to have different colors for different modes, we should use something like this.
```
::selection {
  @apply bg-secondary; // for light mode
}

.dark ::selection {
  @apply bg-primary;; // for dark mode
}
```
or if we want to use only one color for each mode, we can use your solution.
Let me know what you think about it.

Reply:
@yaroslavhoretskyi Ah yes you're right, thanks